### ISSUE TITLE
 How to display the progress of annotations in CVAT, specific to what percentage

### ISSUE THREAD

**Issue by zhanghaihangandliuli** [cvat-ai/cvat#8427]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://docs.cvat.ai/docs/)

### Is your feature request related to a problem? Please describe.

假如我想知道每个标注人员的具体标注进度，要求到百分之几，该如何实现.

### Describe the solution you'd like

假如我想知道每个标注人员的具体标注进度，要求到百分之几，该如何实现.

### Describe alternatives you've considered

假如我想知道每个标注人员的具体标注进度，要求到百分之几，该如何实现.

### Additional context

![image](https://github.com/user-attachments/assets/f27c0bd7-3c67-4c03-adad-1da80a31b763)

**Comment by bsekachev** [comment]
English, please

**Comment by klakhov** [comment]
Hi @zhanghaihangandliuli, 
We display job statuses in task on this progress bar:
![image](https://github.com/user-attachments/assets/b361f9d4-833d-4f21-83d6-e1157c25686d)

The numbers here are controlled by job's `stage`, you can control this value on task page. [Docs](https://docs.cvat.ai/docs/manual/basics/task-details/)

**Comment by zhanghaihangandliuli** [comment]
May I ask if cvat can be used for local secondary development under Windows 10? If so, what tools do I need to download, such as vscode, pyhton3.8, postgreSQL, etc. Do I need to download docker?

**Comment by zhanghaihangandliuli** [comment]
> Hi @zhanghaihangandliuli, 你好 We display job statuses in task on this progress bar:我们在此进度条上的 task 中显示作业状态： ![image](https://private-user-images.githubusercontent.com/50956430/369355773-b361f9d4-833d-4f21-83d6-e1157c25686d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4NDc4ODcsIm5iZiI6MTcyNjg0NzU4NywicGF0aCI6Ii81MDk1NjQzMC8zNjkzNTU3NzMtYjM2MWY5ZDQtODMzZC00ZjIxLTgzZDYtZTExNTdjMjU2ODZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIwVDE1NTMwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZiNDY3YjVkM2Q0NTE0OWIyNjM0MWEwNGM3OThmNTlmNmUzNjU2ZTI0YjE2ZGNlNjMyY2U0M2IxZjQ1NDI3ZWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ZWf89txp9pvv5CLR4Lkqx_wl-GJG-ginlhCxOr5xprw)
> 
> The numbers here are controlled by job's `stage`, you can control this value on task page. [Docs](https://docs.cvat.ai/docs/manual/basics/task-details/)这里的数字由 job `的 stage` 控制，你可以在 task 页面上控制这个值。[文档](https://docs.cvat.ai/docs/manual/basics/task-details/)

May I ask if cvat can be used for local secondary development under Windows 10? If so, what tools do I need to download, such as vscode, pyhton3.8, postgreSQL, etc. Do I need to download docker?

**Comment by klakhov** [comment]
Yes, you can run develempent env on Windows. [Please refer to the documentation](https://docs.cvat.ai/docs/contributing/development-environment/#note-for-windows-users). You will need to install wsl and run all commands from ubuntu section.

**Comment by klakhov** [comment]
I hope all questions regarding this issue have been resolved.
